src/commands/fun/latex.ts

```diff
@@ -0,0 +1,78 @@
+import { ApplicationCommandOptionData, CommandInteraction, MessageEmbed, MessageAttachment } from 'discord.js';
+import fetch from 'node-fetch';
+import { createCanvas, loadImage } from 'canvas';
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = `Accepts a LaTeX expression and posts it as a rendered image.`;
+
+	options: ApplicationCommandOptionData[] = [
+		{
+			name: 'input',
+			description: 'The LaTeX expression to render',
+			type: 'STRING',
+			required: true
+		}
+	]
+
+	async tempRun(interaction: CommandInteraction): Promise<void> {
+		const tex = interaction.options.getString('input');
+		const requestURL = `https://latex.codecogs.com/svg.json?${tex}`;
+		const errorResponse = "Sorry, I couldn't render that LaTeX expression.";
+		let imageAsBase64: Buffer;
+		try {
+			const response = await fetch(requestURL, { method: 'Get' });
+			const imageAsBase64JSON = await response.json();
+			imageAsBase64 = Buffer.from(imageAsBase64JSON.latex.base64, 'base64');
+		} catch (error) {
+			return interaction.reply({ content: errorResponse, ephemeral: true });
+		}
+
+		const image = await loadImage(imageAsBase64);
+
+		// Image will have 4 pixels of padding on all sides
+		const padding = 4;
+		const canvasWidth = image.width + (padding * 2);
+		const canvasHeight = image.height + (padding * 2);
+
+		const canvas = createCanvas(canvasWidth, canvasHeight);
+		const ctx = canvas.getContext('2d');
+		ctx.clearRect(0, 0, canvasWidth, canvasHeight);
+
+		// Invert the default Discord embed background color, entire canvas is inverted later
+		ctx.beginPath();
+		ctx.fillStyle = `rgb(${String(255 - 47)}, ${String(255 - 49)}, ${String(255 - 54)})`;
```

Comment:
I'd rather this rgb string be a constant at the top of the file like: 

`BG_COLOR = `rgb(${255-47}, ${255-49}, ${255-54})`;

You also don't need the `String()` as far as I'm aware, `${}` syntax automatically converts it.